api/model/src/main/java/org/projectnessie/model/Validation.java

```diff
@@ -196,6 +214,26 @@ public static String validateHash(String referenceName) throws IllegalArgumentEx
     throw new IllegalArgumentException(HASH_MESSAGE + " - but was: " + referenceName);
   }
 
+  /** Just checks whether a string is a valid cut-off policy or not. */
+  private static boolean isValidDefaultCutoffPolicy(String policy) {
+    Objects.requireNonNull(policy, "policy must not be null");
+    Matcher matcher = DEFAULT_CUT_OFF_POLICY_PATTERN.matcher(policy);
+    return matcher.matches();
+  }
+
+  /**
+   * Validate default cutoff policy.Policies can be one of: - number of commits as an integer value,
+   * - a duration (see java.time.Duration), - an ISO instant, - 'NONE,' means everything's
+   * considered as live
+   */
+  public static void validateDefaultCutOffPolicy(String value) {
+    if (isValidDefaultCutoffPolicy(value)) {
```

Comment:
I think we can inline `isValidDefaultCutoffPolicy` in this method and no need of private method.